cvat-core/src/annotations-objects.js

```diff
@@ -1215,6 +1300,7 @@
             super(data, clientID, color, injection);
             this.shapeType = ObjectShape.RECTANGLE;
             this.pinned = false;
+            this.annotation_type = data.annotation_type;
```

Comment:
This class is inherited from Annotation  (like the classes below).
So, one such row in the ``Annotations`` constructor would be enough.